docs/content/concepts/assets/asset-selection-syntax.mdx

```diff
@@ -1,32 +1,549 @@
+src="/images/concepts/assets/asset-selection/asset-selection-syntax-dag.png"
+width={2646}
+height={1508}
+/>
+
+### Selecting a single asset
+
+To select a single asset, use the asset's asset key. In this example, we want to select the `taxi_zones_file` asset:
+
+<TabGroup>
+<TabItem name="Python">
+
+```python
+raw_data_job = define_asset_job(name="raw_data_job", selection="taxi_zones_file")
+```
+
+---
+
+</TabItem>
+<TabItem name="CLI">
+
+```shell
+dagster asset list --select taxi_zones_file
+dagster asset materialize --select taxi_zones_file
+```
+
+---
+
+</TabItem>
+<TabItem name="Dagster UI">
+
+```shell
+taxi_zones_file
+```
+
+Which would result in the following asset graph:
+
+<!-- ![Selection query that selects a single asset in the Dagster UI](/images/concepts/assets/asset-selection/select-single-asset.png) -->
+
+<Image
+alt="Selection query that selects a single asset in the Dagster UI"
+src="/images/concepts/assets/asset-selection/select-single-asset.png"
+width={2646}
+height={1508}
+/>
+
+---
+
+</TabItem>
+</TabGroup>
+
+### Selecting assets with multiple key components
+
+To select an asset with a key containing multiple components, such as a prefix, insert slashes (`/`) between the components.
+
+In this example, we want to select the `manhattan/manhattan_stats` asset. The asset is defined as follows - note the `key_prefix`:
+
+```python
+@asset(
+    deps=[AssetKey(["taxi_trips"]), AssetKey(["taxi_zones"])], key_prefix="manhattan"
+)
+def manhattan_stats(database: DuckDBResource):
+ ...
+```
+
+---
+
+<TabGroup>
+<TabItem name="Python">
+
+```python
+manhattan_job = define_asset_job(name="manhattan_job", selection="manhattan/manhattan_stats")
+```
+
+---
+
+</TabItem>
+<TabItem name="CLI">
+
+```shell
+dagster asset list --select manhattan/manhattan_stats
+dagster asset materialize --select manhattan/manhattan_stats
+```
+
+---
+
+</TabItem>
+<TabItem name="Dagster UI">
+
+```shell
+manhattan/manhattan_stats
+```
+
+Which would result in the following asset graph:
+
+<!-- ![Selection query selects the manhattan/manhattan_stats asset in the Dagster UI](/images/concepts/assets/asset-selection/select-multiple-components.png) -->
+
+<Image
+alt="Selection query selects the manhattan/manhattan_stats asset in the Dagster UI"
+src="/images/concepts/assets/asset-selection/select-multiple-components.png"
+width={2646}
+height={1508}
+/>
+
+---
+
+</TabItem>
+</TabGroup>
+
+### Selecting multiple assets
+
+To select multiple assets, use a list of the assets' asset keys. The assets don't have to be dependent on each other.
+
+In this example, we want to select the `taxi_zones_file` and `taxi_trips_file` assets:
+
+<TabGroup>
+<TabItem name="Python">
+
+```python
+raw_data_job = define_asset_job(
+    name="taxi_zones_job", selection=["taxi_zones_file", "taxi_trips_file"]
+)
+```
+
+---
+
+</TabItem>
+<TabItem name="CLI">
+
+```shell
+dagster asset list --select TODO
+dagster asset materialize --select TODO
```

Comment:
I tried doing this a few different ways and couldn't get this to work - am I missing something?